riscv/src/register/mstatus.rs

```diff
@@ -41,6 +41,15 @@ pub enum FS {
     Dirty = 3,
 }
 
+/// Vector extension state
+#[derive(Copy, Clone, Debug, Eq, PartialEq)]
+pub enum VS {
+    Off = 0,
+    Initial = 1,
+    Clean = 2,
+    Dirty = 3,
```

Comment:
Unsure if we want to reuse the `XS` enum instead of creating a brand new enum. However, I'm just following the pattern of `FS` here.